test/util/data_test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import mock
+
+from openhtf.util import data
+
+
+class TestData(unittest.TestCase):
+
+  def test_convert_to_base_types(self):
+    class FloatSubclass(float):
+      pass
+
+    example_data = {
+      'list': [10],
+      'tuple': (10,),
+      'str': '10',
+      'unicode': u'10',
+      'int': 10,
+      'float': 10.0,
+      'long': 10000000000000000000,
```

Comment:
`10000000000000000000` is a long but I can make it `2 ** 80` for clarity

is there a reason I should also check 2 ** 40?